tests/kafkatest/tests/streams/streams_upgrade_test.py

```diff
@@ -474,7 +474,7 @@ def do_rolling_bounce(self, processor, counter, current_generation):
                     monitors[first_other_processor] = first_other_monitor
                     monitors[second_other_processor] = second_other_monitor
 
-                    version_probing_message = "Sent a version 10 subscription and got version 10 assignment back (successful version probing). Downgrade subscription metadata to commonly supported version 10 and trigger new rebalance.",
+                    version_probing_message = "Sent a version 11 subscription and got version 10 assignment back (successful version probing). Downgrade subscription metadata to commonly supported version 10 and trigger new rebalance.",
```

Comment:
When did we bump the number?

Reply:
Here: https://github.com/apache/kafka/pull/10609